geode-core/src/main/java/org/apache/geode/internal/statistics/StatisticsMonitor.java

```diff
@@ -114,14 +100,14 @@ protected void monitor(long millisTimeStamp, List<ResourceInstance> resourceInst
 
   private final void monitorStatisticIds(long millisTimeStamp,
       List<ResourceInstance> resourceInstances) {
-    List<StatisticId> statisticIdsToMonitor = statisticIds;
+    ConcurrentHashSet<StatisticId> statisticIdsToMonitor = statisticIds;
```

Comment:
I don't think this copy is needed since `statisticIdsToMonitor` is never updated after initialization.